lldb/test/API/lang/swift/stepping/TestSwiftStepping.py

```diff
@@ -220,6 +220,7 @@ def do_test(self):
         thread.StepOut()
         self.hit_correct_line(thread, "Second case with a where statement")
 
+        thread.StepOver()
```

Comment:
why the extra StepOver?

Reply:
We now go back to the case once. This might be fixable in the compiler.